src/main/java/com/hubspot/jinjava/lib/filter/LowerFilter.java

```diff
@@ -1,17 +1,17 @@
 /**********************************************************************
-Copyright (c) 2014 HubSpot Inc.
+ Copyright (c) 2014 HubSpot Inc.
```

Comment:
in general, it's nice to separate formatting changes like this out into a separate PR so make your PRs clean and easy to understand.

Reply:
100% agree